Tests/LibWeb/Text/expected/all-window-properties.txt

```diff
@@ -304,6 +304,7 @@ Node
 NodeFilter
 NodeIterator
 NodeList
+Notification
```

Comment:
Oh I forgot to say as well, this change should just be part of the previous commit. That way each commit is "atomic", and tests always pass.